src/main/java/info/sjd/controller/CartController.java

```diff
@@ -65,10 +65,9 @@
 
     @PostMapping(path = "update_status")
     public ResponseEntity<Cart> updateStatus(@RequestParam Integer idParam,@RequestParam Integer closedParam){
-        Cart cart = cartService.updateStatus(idParam, closedParam);
-        if (cart != null){
-            return new ResponseEntity<>(cartService.updateStatus(idParam,closedParam), HttpStatus.OK);
+        if (updateStatus(idParam, closedParam) != null){
+            cartService.updateStatus(idParam,closedParam);
         }
-        return new ResponseEntity<>(HttpStatus.NOT_ACCEPTABLE);
+        return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
You always send OK status. But you need check results before.

Reply:
CarConrollert updateStatus done, and OrderConrollert updateAmount done